tests/perf/vec/box_dyn/src/main.rs

```diff
@@ -0,0 +1,48 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+//! This test checks the performance of pushing onto a vector of Box<dyn _>
```

Comment:
Since nothing actually checks performance, we probably ought to comment this better. Is there an issue to link? Did it completely blow up before? Say so

Reply:
Done.